engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -309,11 +393,19 @@ public void destroyEntitiesUsingPrefab(@CommandParam("prefabName") String prefab
         }
     }
 
+    /**
+     * Exits the game
```

Comment:
This is another good example of javadoc vs description.

* Javadoc: Triggers a graceful shutdown of the game after the current frame, attempting to dispose all game resources
* Description: Closes the game client